### DISCUSSION TITLE
Apollo/Artemis frame rate

### DISCUSSION THREAD

**Discussion by arichiardi** [ClassicOldSong/Apollo#174] [General]
Hi all, I am new here, apologies if this ends up being not the right place.

Just installed Artemis on a 120Hz capable TV and Apollo on a Win 11 machine connected to a 144 HZ capable monitor.

Now, in Sunshine I was able to "advertise" all the refresh rates and so I had 120 in Moonlight. Here in Artemis I only see 30/60 but then when I go to the Virtual Display I definitely can change to 120Hz.

I might be making the classic Fps vs Hz mistake and this is then controlled by each game but just wanted to confirm what I see.

Is 30/60 in Artemis normal? Thank you for your work there, the Virtual Device approach feels like the right way to go!

**Comment by ClassicOldSong** [comment]
The "frame rate advertisement" actually did nothing, and now every client has it removed.

It seems your TV does not report refresh rate correctly to the app. Enable "unlock all refresh rates" in Artemis and try again.

You can verify the actual displayed framerate using UFO test: https://www.testufo.com/frameskipping

**Reply by arichiardi** [reply]
Will try that thank you!

**Comment by arichiardi** [comment]
Ok @ClassicOldSon I trued.

However, the unlock all frane rate option is accessible only via search (not in the main list, unless I am blind).

And even when pressed, the checkbox stays unchecked here.

**Reply by ClassicOldSong** [reply]
It is in the list, almost in the bottom of the list. Once you have checked the box, the settings list should reload.

Can you share what model your TV is?

**Reply by arichiardi** [reply]
Yes it is a Sony Bravia 7 - as an aside, I tried to go to the Server Setting entry on the PC and the brings it to a black screen(I can open a separate issue for that)

**Reply by arichiardi** [reply]
@ClassicOldSong found it and indeed it checked the box and worked from the list (it didn't from the search results - can open an issue with that if you want to track it).

**Reply by ClassicOldSong** [reply]
Server Settings should open the web browser. Probably you don't have a browser installed on your TV?

**Reply by arichiardi** [reply]
Oh a browser on the TV, it might be the case yes

**Comment by arichiardi** [comment]
Well all the issues are solved.

I have to say your work is pretty fantastic, and I really hope you will also find the time for doing what your are enabling others to do - playing some games.

Thank you.
